### ISSUE TITLE
Support HTTP2 option

### ISSUE THREAD

**Issue by nakabonne** [nakabonne/ali#29]
- [ ] Make it possible to switch to send HTTP/2 requests when supported by the server

Only tiny changes are required because all we have to do is just pass to the Vegeta API.

**Comment by shubham14bajpai** [comment]
Hi @nakabonne I would like to work on this. The changes would be as simple as
```go
opts.Attacker = vegeta.NewAttacker(
	vegeta.Timeout(opts.Timeout),
	vegeta.Workers(opts.Workers),
	vegeta.MaxWorkers(opts.MaxWorkers),
	vegeta.MaxBody(opts.MaxBody),
	vegeta.KeepAlive(opts.KeepAlive),
	vegeta.HTTP2(true),
)
```
in the below file. Right? Just wanted to make sure I understand the changes required before raising a PR. https://github.com/nakabonne/ali/blob/fb9a8397e53c8647a3ab6d95d5f8e5c7be3c115f/attacker/attacker.go#L75

**Comment by nakabonne** [comment]
@shubham14bajpai 
>The changes would be as simple as

Exactly. All we have to do is just pass the value received from the flag to vegeta. Look forward to your PR!

**Comment by shubham14bajpai** [comment]
@nakabonne I have raised a PR for the same https://github.com/nakabonne/ali/pull/49

**Comment by nakabonne** [comment]
Closing since this was resolved by #49
